src/mixins/filing-template-mixin.ts

```diff
@@ -164,6 +165,17 @@ export default class FilingTemplateMixin extends Vue {
     return filing
   }
 
+  /**
+   * Builds an Alteration filing body from store data. Used when saving a filing.
+   * @param isDraft boolean indicating whether this is a draft
+   * @returns the Alteration filing body to save
+   */
+  buildAlterationFiling (isDraft: boolean): AlterationFilingIF {
+    // Alteration body to save TBD
+    const alteration: AlterationFilingIF = null
```

Comment:
Or maybe `{}` ?

Reply:
yea :P Sorry this is just a dummy placeholder so i could uphold the return type 😆

Reply:
Actually, `{}` probably won't work anyway as it won't contain the necessary properties for the interface.

Reply:
This is just a placeholder for a future method. I can remove the whole thing if you prefer?

Reply:
No, I like it. This is definitely "framework stuff" that we will expand later.